app/src/main/java/com/kickstarter/libs/KSCurrency.java

```diff
@@ -110,128 +91,88 @@ public String formatWithUserPreference(final double initialValue, final @NonNull
    * Build {@link CurrencyOptions} based on the project and whether we would prefer to show USD. Even if USD is preferred,
```

Comment:
Is this function description still accurate? 🤔

Reply:
was it ever 👀 (😭)

Reply:
i updated it but i think it could be better still